ios/MusicControlManager.m

```diff
@@ -188,6 +188,14 @@ - (void) toggleHandler:(MPRemoteCommand *) command withSelector:(SEL) selector e
     command.enabled = enabled;
 }
 
+- (void) listenToAudioRouteChange:(BOOL) enabled {
```

Comment:
oh yeah the init and dealloc route makes way more sense to me. Made that change and updated the README, let me know if this is all good